templates/python/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    name: test
+    runs-on: ubuntu-latest
```

Comment:
should we use a fixed version here?
wdyt @yannick-roeder

Reply:
Pinned to `ubuntu-22.04`. The latest _release_ image is `ubuntu-18.04`, but currently `ubuntu-latest` is equivalent to the _pre-release_ `ubuntu-22.04`.